src/components/com_cluster/site/controllers/clusterusers.json.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * @package    Com_Cluster
+ * @author     Techjoomla <extensions@techjoomla.com>
+ * @copyright  Copyright (C) 2009 - 2019 Techjoomla. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// No direct access
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Component\ComponentHelper;
+use Joomla\CMS\MVC\Controller\BaseController;
+use Joomla\CMS\Session\Session;
+
+JLoader::import("/components/com_cluster/includes/cluster", JPATH_ADMINISTRATOR);
+
+/**
+ * Get Cluster Users controller class.
+ *
+ * @since  1.0.0
+ */
+class ClusterControllerClusterUsers extends BaseController
+{
+	/**
+	 * Method to get user list depending on the client chosen.
+	 *
+	 * @return   null
+	 *
+	 * @since    1.0.0
+	 */
+	public function getUsersByClientId()
+	{
+		// Check for request forgeries.
+		Session::checkToken() or jexit(JText::_('JINVALID_TOKEN'));
+
+		$clusterIds = Factory::getApplication()->input->getInt('cluster_id', 0);
+		$userOptions = $allUsers = array();
+
+		// Initialize array to store dropdown options
+		$userOptions[] = HTMLHelper::_('select.option', "", Text::_('COM_TJFIELDS_OWNERSHIP_USER'));
```

Comment:
@pravinTek Language constant must be from cluster

Reply:
Used from com_cluster